### PR TITLE
Fix/open close quote

### PR DESCRIPTION
**Pull request by arealclimber**
### DEVELOP

- [x] PositionClosedModal
- [x] PositionOpenModal

### CHECK LIST

- [x]  #847 
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/position_closed_modal/position_closed_modal.tsx [hunk 1]
```diff
@@ -245,13 +245,33 @@ const PositionClosedModal = ({
...
+        data.ticker.split('-')[0] === openCfdDetails.ticker &&
         quotation.data !== null
       ) {
         globalCtx.eliminateToasts(ToastId.GET_QUOTATION_ERROR);
         setQuotationError(false);
         return data;
       } else {
         setQuotationError(true);
+
+        if (data.ticker.split('-')[0] !== openCfdDetails.ticker) {
```
Comment (arealclimber) [inline src/components/position_closed_modal/position_closed_modal.tsx:257]: // TODO: check the unit asset

File: src/components/position_open_modal/position_open_modal.tsx [hunk 2]
```diff
@@ -219,13 +219,34 @@ const PositionOpenModal = ({
...
+        data.ticker.split('-')[0] === openCfdRequest.ticker &&
         quotation.data !== null
       ) {
         globalCtx.eliminateToasts(ToastId.GET_QUOTATION_ERROR);
         setQuotationError(false);
         return data;
       } else {
         setQuotationError(true);
+
+        if (data.ticker.split('-')[0] !== openCfdRequest.ticker) {
```
Comment (arealclimber) [inline src/components/position_open_modal/position_open_modal.tsx:231]: // TODO: check the unit asset

File: src/components/position_open_modal/position_open_modal.tsx [hunk 3]
```diff
@@ -302,6 +323,27 @@ const PositionOpenModal = ({
       return;
     }
 
+    if (openCfdRequest.quotation.ticker.split('-')[0] !== openCfdRequest.ticker) {
```
Comment (arealclimber) [inline src/components/position_open_modal/position_open_modal.tsx:326]: // TODO: check the unit asset

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**